sync2jira/compat.py

```diff
@@ -0,0 +1,14 @@
+
+
+def extract_message_body(msg):
+    """ Strip off the message envelope.
+
+    Handle both fedmsg and fedora-messaging style message bodies.
+    """
+
+    if 'body' in msg:
+        return msg['body']
+    elif 'msg' in msg:
+        return msg['msg']
+    else:
+        raise KeyError(f"Unrecognized message format with keys {msg.keys()}. Expected either 'msg' or 'body'")
```

Comment:
Alternatively,
```suggestion
    body = msg.get("body", msg.get("msg"))
    if body:
        return body
    raise KeyError(f"Unrecognized message format with keys {msg.keys()}. Expected either 'msg' or 'body'")
```
This avoids duplicate key look-ups, and it also raises an error if the value exists but is empty.  (If you don't like this last part, you can change the `if` test to check for `None` explicitly to allow it to return an empty body.)

Reply:
This is a level of optimization I almost never think about. Thank you for the suggestion!